draft-ietf-add-split-horizon-authority.xml

```diff
@@ -584,10 +617,17 @@
       However, security by obscurity may not maintain or increase the security of the
       internal domain names, as they may be leaked in various other ways
       (e.g., browser reload).</t>
+      <t>The Authentication Domain Names of authorized local encrypted resolvers are
```

Comment:
The certificate transparency log would anyway reveal the Authentication Domain Name and this draft is not creating a new privacy issue.

Reply:
I agree.  This is covered in the preceding paragraph.